sqlmesh/utils/pydantic.py

```diff
@@ -27,14 +27,10 @@
 
 
 def field_validator(*args: t.Any, **kwargs: t.Any) -> t.Callable[[t.Any], t.Any]:
```

Comment:
I tried to import these validators instead of overriding them, which resulted in several mypy errors that were previously silenced due to the `t.Any` hints. For this reason I decided to keep them, at least temporarily. Fixing the mypy issue was a little annoying.